salt/edx/files/nginx_static_assets.j2

```diff
@@ -0,0 +1,20 @@
+  location ~ /static/((?P<dir>[^/]+)/(?P<file>.*)|(?P<rfile>[\w\d\-.]+)) {
+    root {{ edxapp_app_dir }};
+    try_files /staticfiles/$dir/$file /course_static/$dir/static/$file /staticfiles/$rfile =404;
```

Comment:
This line needs to have `/prod_course_static/$dir/static/$file` if it's a live instance, and `/course_static/$dir/static/$file` if it's a draft instance.

In fact, we can get away with one symlink here and have it point to the right place. I will leave another comment with details.